<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>911</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>API test</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(135, 213, 255);

</string>
      </property>
      <property name="title">
       <string>2-&gt;Mode</string>
      </property>
      <widget class="QPushButton" name="self_button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Self-Driving</string>
       </property>
      </widget>
      <widget class="QPushButton" name="manual_button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>101</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Manual</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_mode">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Unknow</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>191</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>3-&gt;Steer</string>
      </property>
      <widget class="QPushButton" name="steerEnableButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
      <widget class="QPushButton" name="steerDisableButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Disable</string>
       </property>
      </widget>
      <widget class="QPushButton" name="steerSetButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>41</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QPushButton" name="steerAddButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="steerSubButton">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="objSteer">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>steer:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nowSteer">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>31</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_steerStatus">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>191</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>5-&gt;Acceleration</string>
      </property>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>91</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Disable</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>41</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_10">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>31</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>71</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>31</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>acc:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nowAcc">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_accStatus">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>121</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 255, 255);</string>
      </property>
      <property name="title">
       <string>1-&gt;Service</string>
      </property>
      <widget class="QPushButton" name="InitSButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>InitService</string>
       </property>
      </widget>
      <widget class="QPushButton" name="unintServiceButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>UnintService</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>141</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>4-&gt;Gear</string>
      </property>
      <widget class="QPushButton" name="setNGearButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>N</string>
       </property>
      </widget>
      <widget class="QPushButton" name="setDGearButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>D</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nowGear">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>N</string>
       </property>
      </widget>
      <widget class="QPushButton" name="setRGearButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>R</string>
       </property>
      </widget>
      <widget class="QPushButton" name="setPGearButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>P</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>241</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>6-&gt;Park &amp;&amp; Stop</string>
      </property>
      <widget class="QPushButton" name="pushButton_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Press</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_16">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Release</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_17">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>151</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>slow down to stop</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>9</y>
        <width>181</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
font: 57 italic 11pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="title">
       <string>Status</string>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Speed(km/h):</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Battery(V):</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>engine:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>BrakeOpening:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>131</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>ThrottleOpening:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nowSpeed">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nowBattery">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nowEngine">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>90</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nowBOpening">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_nowTOpening">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>150</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>ConnectStatus</string>
      </property>
      <widget class="QLabel" name="label_connectStatus">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 57 italic 16pt &quot;Ubuntu&quot;;</string>
       </property>
       <property name="text">
        <string>offline</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>391</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>测试刹车相关功能</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>521</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>441</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Copyright(c) 2018 CQJD XINGYUAN GROUP. All rights reserved. </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_nowTime">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>319</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 oblique 12pt &quot;Waree&quot;;
gridline-color: rgb(255, 85, 255);</string>
      </property>
      <property name="text">
       <string>now Time</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>9</y>
        <width>181</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="title">
       <string>Status</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>fun2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>fun3</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page4</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuA">
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QMenu" name="menuB">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menuA"/>
   <addaction name="menuB"/>
  </widget>
  <action name="action2">
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action1_1">
   <property name="text">
    <string>1.1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
